yw_website/yw_website/apps/website/templates/includeable/workflow_list.html

```diff
@@ -26,37 +25,47 @@
 
         {% for workflow in workflow_list %}
             <li>
-                <div classs = "tags">
-                    Tags: {{ workflow.tags | join:", " }}
-                </div>
-                <div class = "title">
-                    {{ workflow.title|linebreaks }}
-                </div>
-                <div class = "last_modified"> 
+                <a class = "title" href="{% url 'detailed_workflow' workflow.id workflow.version_id %}"">
+                        {{ workflow.title|linebreaks }}
+                </a>
+                <div class = "last-modified"> 
                     {{ workflow.version_modified }}
                 </div>
-
-                <div id="graph{{ forloop.counter }}" class = workflow-image></div>
-
-                <div class = "workflow-details" id="description">
+                
+                <div  class = "workflow-description" id="description">
                     <b>Description: </b>{{ workflow.description }}
                 </div>
 
-                <div style = "text-align:center">
-                    <a class = "more-details-button" href="{% url 'detailed_workflow' workflow.id workflow.version_id %}"> View Details</a>
-                </div>      
-                <!-- <button class="details-button" onClick = "description_view()">View Description</button> -->
+                <div >                
+                    <input id = "description-button" type="button" value = "Read More..." class = "expand-description-button" onClick="description_view()" ></input>
+                </div>
+                <div class = "tags-label">
+                        Tags:
+                        {% for tag in workflow.tags %}
+                        <div class = "tags">
+                            <a id = "tag" href="{% url 'haystack_search'%}">{{ tag }}</a>
+                        </div>
+                        {% endfor %}
+                    </div>
+
             </li>
             <script>
                 function description_view(){
                     var description = document.getElementById("description");
-                    if (description.style.display === "none") {
-                        description.style.display = "block";
+                    var button = document.getElementById("description-button")
+                    if (window.getComputedStyle(description, null).getPropertyValue("height") === '60px') {
```

Comment:
This doesn't work, and I'm not too sure it's even worth adding

Reply:
Also the same javascript is applied to each element

Reply:
>I'm not too sure it's even worth adding

I think it is worth adding